eng/pipelines/templates/steps/analyze.yml

```diff
@@ -110,6 +110,14 @@ steps:
     failOnStderr: true
     workingDirectory: 'sdk/${{parameters.ServiceDirectory}}'
 
+  - task: PowerShell@2
```

Comment:
@Alancere @raych1 what is the context for this change? I don't see any description in the PR so I'm not sure why we are doing this.

Reply:
@weshaggard , I added more info to the description.

Reply:
Even if we go with this solution this is essentially always ignoring this link which we don't want to do. At what point would we ever validate it?

While this is an interesting idea I'm not convinced it is the best option here.